src/foundry/client/pixi/groups/effects.d.ts

```diff
@@ -1,29 +1,149 @@
+// FOUNDRY_VERSION: 10.291
+
 /**
  * A container group which contains visual effects rendered above the primary group.
+ *
+ * ### Hook Events
+ * - {@link hookEvents.drawEffectsCanvasGroup}
+ * - {@link hookEvents.createEffectsCanvasGroup}
+ * - {@link hookEvents.lightingRefresh}
  */
 declare class EffectsCanvasGroup extends PIXI.Container {
   constructor();
 
-  walls: WallsLayer;
+  /**
+   * The current global light source.
+   */
+  globalLightSource: LightSource;
+
+  /**
+   * Whether to currently animate light sources.
+   * @defaultValue `true`
+   */
+  animateLightSources: boolean;
+
+  /**
+   * Whether to currently animate vision sources.
+   * @defaultValue `true`
+   */
+  animateVisionSources: boolean;
+
+  /**
+   * A mapping of light sources which are active within the rendered Scene.
+   * @defaultValue `new foundry.utils.Collection<LightSource>()`
+   */
+  lightSources: Collection<LightSource>;
+
+  /**
+   * A Collection of vision sources which are currently active within the rendered Scene.
+   * @defaultValue `new foundry.utils.Collection<VisionSource>()`
+   */
+  visionSources: Collection<VisionSource>;
+
+  /**
+   * Create the child layers of the effects group.
+   */
+  #createLayers(): void;
+
+  /**
+   * Clear all effects containers and animated sources.
+   */
+  clearEffects(): void;
+
+  /**
+   * Draw the component layers of the canvas group.
+   */
+  draw(): Promise<void>;
+
+  /**
+   * Initialize LightSource objects for all AmbientLightDocument instances which exist within the active Scene.
+   */
+  initializeLightSources(): void;
+
+  /**
+   * Update the global light source which provide global illumination to the Scene.
+   */
+  protected _updateGlobalLightSource(): GlobalLightSource;
```

Comment:
`GlobalLightSource` will show a type error until #2072 is completed.

Reply:
That's fine. If we really wanted, we could just add a `type GlobalLightSource = undefined` to the top level of the file to suppress the error, but that's not 100% needed.